data/json/martialarts.json

```
@@ -70,6 +70,53 @@
     ],
     "weapons": [ "bagh_nakha", "bio_claws_weapon", "cestus", "knuckle_brass", "knuckle_nail", "knuckle_steel", "knuckle_steel_forged" ]
   },
+  {
+    "type": "martial_art",
+    "id": "style_bojutsu",
+    "name": { "str": "Bōjutsu" },
+    "description": "Bōjutsu, \"The Way of the Staff\", is the Japanese martial art of fighting with a staff. Bōjutsu focuses on maintaining a good stance to effectivity block and counterattack.",
+    "initiate": [ "You position yourself into an agile stance.", "%s enters an agile stance." ],
+    "learn_difficulty": 5,
+    "primary_skill": "bashing",
+    "strictly_melee": true,
+    "static_buffs": [
+      {
+        "id": "buff_bojutsu_static",
+        "name": "Bōjutsu Stance",
+        "description": "A agile stance that allows your blocks to reduce incoming damage.\n\nBlocked damage reduced by 100% of Dexterity.",
+        "melee_allowed": true,
+        "flat_bonuses": [ { "stat": "block", "scaling-stat": "dex", "scale": 1 } ]
+      },
+      {
+        "id": "buff_bojutsu_static2",
+        "name": "Skilled Striking",
+        "description": "As you get more skilled you're able to begin targeting striking points that allow more for damage.\n\nMelee skill increases Bash Weapon Damage Cap.",
+        "melee_allowed": true,
+        "melee_bash_damage_cap_bonus": true,
+        "skill_requirements": [ { "name": "melee", "level": 1 } ]
+      }
+    ],
+    "onmove_buffs": [
+      {
+        "id": "buff_bojutsu_onmove",
+        "name": "Rolling Staff",
+        "description": "As you move you begin to rapidly and fluidly roll your staff between your hands.\n\n+10% Bash damage.\nEnables \"Rolling Strike\" technique.\nLasts for 1 turn.",
+        "skill_requirements": [ { "name": "melee", "level": 3 } ],
+        "melee_allowed": true,
+        "buff_duration": 1,
+        "mult_bonuses": [ { "stat": "damage", "type": "bash", "scale": 1.1 } ]
+      }
+    ],
+    "techniques": [
+      "tec_bojutsu_thrust",
+      "tec_bojutsu_kneestrike",
+      "tec_bojutsu_frontstrike",
+      "tec_bojutsu_rollingstrike",
+      "tec_bojutsu_feint",
+      "tec_bojutsu_disarm"
+    ],
+    "weapons": [ "stick_long", "l-stick", "long_pole", "q_staff", "i_staff", "shock_staff", "broom", "mop" ]
```

I'm not sure if the long stick and long pole should be usable by the martial art, since both have a length more in line with the spears and similar polearms, if you are still working on this, I suggest just using the: `"weapon_category": [ "QUARTERSTAVES" ],` since it already includes all the quarterstaves, the other weapons only need to have the category added to them. 
```suggestion
    "weapon_category": [ "QUARTERSTAVES" ],
    "weapons": [ "stick_long", "long_pole", "broom", "mop" ]
```